app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx

```diff
@@ -19,6 +19,8 @@ import { connect } from 'react-redux';
 import { injectIntl, defineMessages } from 'react-intl';
 import PropTypes from 'prop-types';
 import { commonValidators } from 'common/utils/validation';
+import * as validate from 'common/utils/validation/validate';
+import { bindMessageToValidator } from 'common/utils/validation/validatorHelpers';
```

Comment:
You can simply import everything related to validation from one source:
`import { bindMessageToValidator, validate } from 'common/utils/validation';`